src/main/java/io/github/cottonmc/energy/api/DefaultEnergyTypes.java

```diff
@@ -0,0 +1,43 @@
+package io.github.cottonmc.energy.api;
+
+import io.github.cottonmc.energy.CottonEnergy;
+import net.minecraft.util.Identifier;
+
+public class DefaultEnergyTypes {
+
+	public static final EnergyType ENERGY_TYPE_EMPTY = register("empty", new EmptyEnergyType());
```

Comment:
`= register("cotton:empty", new EmptyEnergyType());`

Reply:
Doesn't need the namespace, it runs through the same `register` method.